spec/consensus/reactor/README.md

```diff
@@ -0,0 +1,113 @@
+# Consensus Reactor
+
+CometBFT is a state machine replication framework and part of the stack used in the Cosmos ecosystem to build distributed applications, such as the Cosmos SDK.
```

Comment:
I wouldn't define the Cosmos SDK as an app. Consider mentioning the Cosmos Hub instead